src/node/client_ui_interface.h

```diff
@@ -1,11 +1,11 @@
 // Copyright (c) 2010 Satoshi Nakamoto
 // Copyright (c) 2012-2020 The Bitcoin developers
-// Copyright (c) 2014-2022 The Gridcoin developers
+// Copyright (c) 2014-2023 The Gridcoin developers
 // Distributed under the MIT/X11 software license, see the accompanying
 // file COPYING or https://opensource.org/licenses/mit-license.php.
 
-#ifndef BITCOIN_NODE_UI_INTERFACE_H
-#define BITCOIN_NODE_UI_INTERFACE_H
+#ifndef BITCOIN_NODE_CLIENT_UI_INTERFACE_H
+#define BITCOIN_NODE_CLIENT_UI_INTERFACE_H
```

Comment:
Why the rename of these two files, which requires a bunch of include changes across many other files? Is there some sort of conflict generated by using cmake?

Reply:
Yes, AutoUIC tries to process all files starting with "ui_"

Reply:
ok